.github/actions/setup_dbt/action.yaml

```diff
@@ -4,6 +4,10 @@ inputs:
   role-to-assume:
     description: AWS IAM role to assume when running dbt operations.
     required: true
+  role-duration-seconds:
+    description: Expiration time for AWS OIDC token. Default is one hour.
+    required: false
+    default: 3600
```

Comment:
Now that the maximum expected build time is [about four hours](https://github.com/ccao-data/data-architecture/actions/runs/6086810837) in the worst case where all resources need to be rebuilt, we have to bump the expiration time for the OIDC token, which requires passing this variable through to the underlying `configure-aws-credentials` action.